lightning/src/ln/peer_handler.rs

```diff
@@ -627,12 +627,12 @@ pub type SimpleArcPeerManager<SD, M, T, F, C, L> = PeerManager<
 ///
 /// This is not exported to bindings users as general type aliases don't make sense in bindings.
 pub type SimpleRefPeerManager<
-	'a, 'b, 'c, 'd, 'e, 'f, 'g, 'h, 'i, 'j, 'k, 'l, 'm, 'n, SD, M, T, F, C, L
+	'a, 'b, 'c, 'd, 'e, 'f, 'g, 'h, 'i, 'j, SD, M, T, F, C, L
 > = PeerManager<
 	SD,
-	&'n SimpleRefChannelManager<'a, 'b, 'c, 'd, 'e, 'f, 'g, 'm, M, T, F, L>,
-	&'f P2PGossipSync<&'g NetworkGraph<&'f L>, &'h C, &'f L>,
-	&'i SimpleRefOnionMessenger<'g, 'm, 'n, L>,
+	&'j SimpleRefChannelManager<'a, 'b, 'c, 'd, 'e, 'f, 'g, 'i, M, T, F, L>,
+	&'f P2PGossipSync<&'g NetworkGraph<&'f L>, C, &'f L>,
```

Comment:
Looks preexisting, but should we use different lifetime than `'f` for `P2PGossipSync` given it uses that lifetime for its parameters?